src/backend/core/tests/test_api_stats.py

```diff
@@ -30,7 +30,7 @@ def test_api_stats__anonymous(django_assert_num_queries):
     assert response.json() == {
         "total_users": 0,
         "mau": 0,
-        "domains": 5,
+        "active domains": 5,
```

Comment:
Consistency objection: the resulting code spells "total users" with an underscore and "active domains" with a space.

Spaces are more legible, and can make parsing harder from some oddball languages. Underscores are more portable but less aesthetic.

Reply:
Damn you're right 🤦🏽 seems like I can't focus these days. I'll go for underscores as aesthetic doesn't matter much here.